### ISSUE TITLE
FFTW3LibraryDepends.cmake referenced but doesn't exist...?

### ISSUE THREAD

**Issue by 20DM** [FFTW/fftw3#332]
Hi,

the CMake config file appears to reference a `FFTW3LibraryDepends.cmake` which doesn't seem to exist:

https://github.com/FFTW/fftw3/blob/38ea230e25e69e7a3f35b957b815bac4f9aa22b0/FFTW3Config.cmake.in#L13

Is this by accident or am I missing something...?

Cheers,
Chris

**Comment by stevengj** [comment]
It's generated by the rules in CMakelists.txt when you run `cmake`

**Comment by 20DM** [comment]
.. but not when you build with autotools, which is the default. The builds are inconsistent currently as far as I can tell.

**Comment by stevengj** [comment]
Yes, it might be better just to omit the `FFTW3Config.cmake.in` installation from the autotools build completely.

**Comment by 20DM** [comment]
For what it's worth, when I build FFTW with autotools (or using Homebrew) and then manually remove that include line from the `FFTW3Config.cmake`, I can still link against the package just fine with my separate CMake-based application.
